submissions/israelOladosu/database/transportation.sql

```diff
@@ -12,44 +12,33 @@ CREATE TABLE classes (
 -- passangers table
 CREATE TABLE passengers (
     id INT NOT NULL AUTO_INCREMENT,
-    firstname VARCHAR(50) NOT NULL,
-    lastname VARCHAR(50) NOT NULL,
+    name VARCHAR(50) NOT NULL,
     sex VARCHAR(6) NOT NULL,
     age INT,
     PRIMARY KEY(id)
 );
 
 -- trips table
 CREATE TABLE trips (
-    id INT NOT NULL AUTO_INCREMENT,
```

Comment:
This table should have an AUTO_INCREMENT `id` which will be the PRIMARY KEY

Reply:
And not `passenger_id` as the PRIMARY_KEY

Reply:
changes effected